.github/workflows/prepare-for-prod-ut-deploy.yml

```diff
@@ -134,3 +136,39 @@ jobs:
           git push -u origin hosted-user-transformer-$UT_TAG_NAME
 
           gh pr create --fill
+
```

Comment:
I'm not sure whether DM team wants to track these every release - otherwise PR's are gonna be in hanging state cc: @Jayachand

Reply:
user-transformers have special needs to put on dedicated

Reply:
cc: @abhimanyubabbar

Reply:
Are we saying to deploy UT to production customers ( shared ones ) as well through a manual trigger ?

Reply:
**Only** move production UT deployment to a manual trigger.